tests/Specs/New.cs

```diff
@@ -75,6 +75,46 @@ public void Business(BaseLayer layer)
             });
         }
 
+        [Theory]
+        [InlineData(BaseLayer.Core, RepositoryType.App)]
+        [InlineData(BaseLayer.MES, RepositoryType.Customization)]
+        public void Grafana(BaseLayer baseLayer, RepositoryType repositoryType)
+        {
+            RunNew(
+                new GrafanaCommand(), 
+                "Cmf.Custom.Grafana",
+                mesVersion: "10.2.1", 
+                ngxSchematicsVersion: "1.3.3",
+                baseLayer: baseLayer, 
+                repositoryType: repositoryType, 
+                extraAsserts: args =>
+            {
+                var (pkgVersion, dir) = args;
+                Assert.True(Directory.Exists("Cmf.Custom.Grafana"), "Grafana package folder is missing");
+                Assert.True(Directory.Exists($"Cmf.Custom.Grafana/{pkgVersion}"), "Grafana Version folder is missing");
+                Assert.True(Directory.Exists($"Cmf.Custom.Grafana/{pkgVersion}/dashboards"), "Grafana dashboards folder is missing");
+                Assert.True(Directory.Exists($"Cmf.Custom.Grafana/{pkgVersion}/datasources"), "Grafana datasources folder is missing");
+
+                Assert.True(Directory.Exists($"Cmf.Custom.Grafana/{pkgVersion}/dashboards/CF58AABF/"), "Project Dashboards folder is missing or has wrong name");
```

Comment:
the CF58AABF is random, you need to fetch the name from the base method (there are some examples in the tests, IIRC)